FirebaseAI/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [added] Added support for Grounding with Google Search. (#15014)
```

Comment:
When fixing the merge conflict in this file, let's keep this addition on the top.